ansible/roles/operator-pipeline/templates/openshift/tasks/merge-registry-credentials.yml

```diff
@@ -37,8 +53,7 @@ spec:
           echo "Project contains secret registry tokens. Merging with internal tokens..."
           echo $PROJECT_CONFIG_JSON | sed 's/\\n/\n/g' > project_dockerconfigjson.gpg
 
-          gpg -v --batch --import <(cat "$(workspaces.gpg-key.path)/exd-guild-isv.key")
-          GPG_PASSPHRASE=`(cat "$(workspaces.gpg-key.path)/exd-guild-isv.passphrase")`
+          gpg -v --batch --import <$GPG_KEY
```

Comment:
@Allda I had a bit of trouble wrapping my head about the original line (`gpg -v --batch --import <(cat "$(workspaces.gpg-key.path)/exd-guild-isv.key")`, could you please take a look at this and verify that it's updated correctly? Thanks.

Reply:
I think it looks ok. Did you verify it works?

Reply:
So it turned out it didn't work :( I updated it to a format much closer to the original, and it ran okay.